encrypted-media/clearkey-events.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Events with Clear Key</title>
+    <link rel="help" href="https://w3c.github.io/encrypted-media/">
+
+    <!-- Web Platform Test Harness scripts -->
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+
+    <!-- Helper scripts for Encrypted Media Extensions tests  -->
+    <script src=/encrypted-media/util/utils.js></script>
+    <script src=/encrypted-media/util/utf8.js></script>
+    <script src=/encrypted-media/util/fetch.js></script>
+
+    <!-- Content metadata -->
+    <script src=/encrypted-media/content/content-metadata.js></script>
+
+    <!-- Message handler for Clear Key -->
+    <script src=/encrypted-media/util/clearkey-messagehandler.js></script>
+
+    <!-- The script for this specific test -->
+    <script src=/encrypted-media/scripts/events.js></script>
+
+  </head>
+  <body>
+    <div id='log'></div>
+
+    <script>
+        getSupportedContent( 'org.w3.clearkey' )
+        .then( function( contents )
+        {
+            runTest( {  keysystem:          'org.w3.clearkey',
+                        content:            contents[ 0 ],
+                        messagehandler:     messagehandler.bind( contents[ 0 ] ),
+                     } );
+        } );
+    </script>
+  </body>
+</html>
```

Comment:
Yes, the script associated with this was based on encrypted-media-events. The idea is that the new scripts in the scripts folder can be called from multiple HTML files so that we can easily get { clear key, drm } x { mp4, webm } where the Google tests were just clear key + webm.